pkg/domain/attributor.go

```diff
@@ -37,3 +38,49 @@ type NexposeAttributedAssetVulnerabilities struct {
 	Vulnerabilities []AssetVulnerabilityDetails `json:"assetVulnerabilityDetails"`
 	BusinessContext CloudAssetDetails           `json:"businessContext"`
 }
+
+// AssetNotFoundError occurs when a request to an asset inventory system
+// returns either a 404 Not Found response, or a 200 OK response with no results
```

Comment:
Question:
> or a 200 OK response with no results
In what case would the asset-attribution-api return a 200 OK, but not return any results?

Reply:
`asset-inventory-api` should never return a 200 with no results, but I believe the assumption is that other inventory systems might.

Reply:
Got it, thanks

Reply:
yep, that was the idea.